fdbrpc/sim2.actor.cpp

```diff
@@ -756,7 +756,7 @@ class Sim2 : public ISimulator, public INetworkConnections {
 
 	// timer() can be up to one second ahead of now()
 	virtual double timer() {
-		timerTime += deterministicRandom()->random01()*(time+1.0-timerTime)/2.0;
+		timerTime += deterministicRandom()->random01()*(time+0.1-timerTime)/2.0;
```

Comment:
Does this require an update to the comment above as well?

Reply:
good catch